Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1980,10 +1987,13 @@ extension BrowserViewController: TabDelegate {
     readerMode.delegate = self
     tab.addContentScript(readerMode, name: ReaderMode.name(), contentWorld: .defaultClient)
 
-    // only add the logins helper if the tab is not a private browsing tab
+        // only add the logins helper and wallet provider if the tab is not a private browsing tab
     if !tab.isPrivate {
       let logins = LoginsHelper(tab: tab, profile: profile, passwordAPI: braveCore.passwordAPI)
       tab.addContentScript(logins, name: LoginsHelper.name(), contentWorld: .defaultClient)
+      #if WALLET_DAPPS_ENABLED
```

Comment:
We usually put such flags as a negation and versioned them,
does not matter for me personally as this flag will be dropped really soon